src/main/java/seedu/address/model/Model.java

```diff
@@ -76,6 +77,10 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
+    void addEvent(Event event);
+
+    void deleteEvent(Event event);
+
```

Comment:
Need to add the respective comments to these abstract methods.
```suggestion
    /**
     * Adds the given event.
     * TODO: {@code event} must not already exist in the address book.
     */
    void addEvent(Event event);
    
    /**
     * Deletes the given event.
     * The event must exist in the address book.
     */
    void deleteEvent(Event event);

```

Reply:
ok, thanks for the suggestions